hotfix(a11y): fix for aria-hidden not applying to slides properly

pimdewit opened the pull request:
https://github.com/ciampo/macro-carousel/issues/43
Turns out that `inert-polyfill` (used by the demos) has a side-effect if you apply it by calling `setAttribute('inert', true)`. It takes a frame before aria-hidden is applied. You can omit this by setting inert the "native" way; `element.inert = flag`.

This is not a bug on our end, but it looks like this fix would not hurt

ciampo commented on src/macro-carousel/macro-carousel.js:
```diff
@@ -1145,15 +1145,15 @@ value. Add CSS units to its value to avoid breaking the slides layout.`);
-        slideEl.setAttribute(ATTRS.STANDARD.TABINDEX, -1);
-      } else {
-        slideEl.setAttribute(ATTRS.STANDARD.INERT, '');
+        slideEl.setAttribute(ATTRS.STANDARD.TABINDEX, '-1');
```
Judging at the comment just before the `if`, it looks like I left a bug in those lines.
I think, if `isSlideInView` is `true`, the `tabindex` attribute should be removed (instead of being set to `-1`)

pimdewit replied:
I think its currently working as intended, we want to give `tabindex -1` so we can programmatically focus the slide right?

[https://mzl.la/2LY933V](https://mzl.la/2LY933V)
<img width="540" alt="Screenshot 2019-10-08 at 18 01 48" src="https://user-images.githubusercontent.com/1285982/66416486-d0bfc980-e9f5-11e9-9d25-63e9d66e28d9.png">

pimdewit replied:
`wicg-inert` seems to do some magic with tabindex too.. 😕

pimdewit commented:
I've updated the PR - we set the tabindex in `_onSlidesSlotChange` to -1 to ensure we can programmatically focus the slides.
If `inert=true`, the element will not be focusable regardless of the tabindex value. It will also set the semantic properties under the hood, including aria-hidden (and this is reflected through an attribute when using the `wicg-inert` polyfill).

So basically I have removed the aria-hidden and tabindex logic, since `inert` takes care of that. What do you think @ciampo?

ciampo commented:
What happens now though is:

- setting the `inert` property to `true`:
  - adds the `inert` and `aria-hidden="true"` attributes
  - removes the `tabindex` attribute
- setting the `inert` property to `false`:
  - removes the `inert` and `aria-hidden` attributes
  - adds the `tabindex="-1"` attribute


This logic conflicts with the slides having `tabindex="-1"` set in the `_onSlidesSlotChange` function — we should probably remove this bit of code and handle this accessibility aspect all through the `inert` property?

Do these changes affect at all the bit where the newly selected slide is programmatically focused?
